This relaxes the comparison threshold between the reference data
and the bitmaps produced by Ghostscript interpreting the vectorized
image.  Apparently the threshold has become too low with the switch
to from Ghostscript 2.21 to 2.22.

--- potrace-1.15/check/postscript-check.sh	2018-01-06 19:47:26.194900686 +0100
+++ potrace-1.15/check/postscript-check.sh	2018-01-06 19:51:35.937503739 +0100
@@ -70,15 +70,15 @@ if test $? -ne 0; then
     echo "Something is wrong with $GS; skipping this test" >& 2
     exit 77
 fi 
-actiondiff "$TMP2" "$REFDATA" 1000
+actiondiff "$TMP2" "$REFDATA" 1500
 
 action $POTRACE -r50 -p -L 0 -B 0 --opaque -o "$TMP1" "$DATA"
 action "$GS" -q -dNOPAUSE -sDEVICE=pbmraw -g460x394 -r100x100 -sOutputFile="$TMP2" -- "$TMP1"
-actiondiff "$TMP2" "$REFDATA" 1200
+actiondiff "$TMP2" "$REFDATA" 1500
 
 action $POTRACE -r50 -p -L 0 -B 0 -A 160 -o "$TMP1" "$DATA"
 action "$GS" -q -dNOPAUSE -sDEVICE=pbmraw -g568x528 -r100x100 -sOutputFile="$TMP2" -- "$TMP1"
-actiondiff "$TMP2" "$REFDATAROT" 1200
+actiondiff "$TMP2" "$REFDATAROT" 1500
 
 action rm -f "$TMP1"
 action rm -f "$TMP2"

--- potrace-1.15/check/pdf-check.sh	2018-01-06 19:47:28.698886663 +0100
+++ potrace-1.15/check/pdf-check.sh	2018-01-06 20:54:14.895460329 +0100
@@ -72,15 +72,15 @@ if test $? -ne 0 -o ! -f "$TMP2"; then
     echo "GS does not understand PDF; skipping this test" >& 2
     exit 77
 fi 
-actiondiff "$TMP2" "$REFDATA" 1000
+actiondiff "$TMP2" "$REFDATA" 1500
 
 action $POTRACE -r50 -p -L 0 -B 0 --opaque -b pdf -o "$TMP1" "$DATA"
 action "$GS" -q -dNOPAUSE -sDEVICE=pbmraw -g460x394 -r100x100 -sOutputFile="$TMP2" -- "$TMP1"
-actiondiff "$TMP2" "$REFDATA" 1200
+actiondiff "$TMP2" "$REFDATA" 1500
 
 action $POTRACE -r50 -p -L 0 -B 0 -A 160 -b pdf -o "$TMP1" "$DATA"
 action "$GS" -q -dNOPAUSE -sDEVICE=pbmraw -g568x528 -r100x100 -sOutputFile="$TMP2" -- "$TMP1"
-actiondiff "$TMP2" "$REFDATAROT" 1200
+actiondiff "$TMP2" "$REFDATAROT" 1500
 
 action rm -f "$TMP1"
 action rm -f "$TMP2"

Diff finished.  Sat Jan  6 20:54:50 2018
